oe_theme.theme

```diff
@@ -127,6 +127,27 @@ function oe_theme_preprocess_paragraph__oe_accordion(array &$variables): void {
   }
 }
 
+/**
+ * Implements hook_preprocess_paragraph__oe_contextual_navigation().
+ */
+function oe_theme_preprocess_paragraph__oe_contextual_navigation(array &$variables): void {
+  // Massage data to be compliant with ECL 'context_nav' pattern data structure.
+  /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
+  $paragraph = $variables['paragraph'];
+
+  $variables['label'] = $paragraph->get('field_oe_title')->value;
```

Comment:
Label is never used, as inside the template we use directly `content.field_oe_title`.